assets/i18n/en.json

```diff
@@ -5,7 +5,7 @@
     "I18N_ABOUT_PAGE_ABOUT_TAB_FEEDBACK_TEXT": "to improve your exploration.",
     "I18N_ABOUT_PAGE_ABOUT_TAB_HEADING": "About Oppia",
     "I18N_ABOUT_PAGE_ABOUT_TAB_PARAGRAPH_1": "Oppia's mission is to help anyone learn anything they want in an effective and enjoyable way.",
-    "I18N_ABOUT_PAGE_ABOUT_TAB_PARAGRAPH_2": "By creating a set of free, high-quality, demonstrably effective lessons with the help of educators from around the world, Oppia aims to provide students with quality education &mdash; regardless of where they are or what traditional resources they have access to.",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_PARAGRAPH_2": "By creating a set of free, high-quality and effective lessons with the help of educators from around the world, Oppia aims to provide students with quality education &mdash; regardless of where they are or what traditional resources they have access to.",
```

Comment:
Is this an expected change for this PR?

Reply:
No @kevintab95 This was a small change just to filter out the exact error i was receiving.
This is not an expected change

Reply:
I have made the required changes in he en.json file.

Reply:
Cool, thanks!